### ISSUE TITLE
Could TensorFlow support tf.nn.separable_conv1d in future release?

### ISSUE THREAD

**Issue by mingyr** [tensorflow/tensorflow#19247]
Dear TensorFlower:

Currently TensorFlow only has tf.nn.separable_conv2d support and it is convenient to use.

Could it is possible in future release tf.nn.separable_conv1d be added? Since I am intensively using TensorFlow for processing bio-signals, so it is more attractive for TensorFlow having such support at this regard.

Best regards

The following fields are filled due to request, since it is not a issue but more like a feature, not sure the information below is helpful or not.

Have I written custom code: No
OS Platform and Distribution: RedHat 7.3
TensorFlow installed from: Customer build from source code
TensorFlow version: 1.8
Bazel version: 0.12
CUDA/cuDNN version:9.1/7.0.5
GPU model and memory: P5000/16G
Exact command to reproduce: N/A

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
Have I written custom code
OS Platform and Distribution
TensorFlow installed from
TensorFlow version
Bazel version
CUDA/cuDNN version
GPU model and memory
Exact command to reproduce

**Comment by tensorflowbutler** [comment]
It has been 14 days with no activity and the `awaiting response` label was assigned. Is this still an issue?

**Comment by mingyr** [comment]
Yes, the required information has been provided in the original post, however it is posted here again:

Have I written custom code: No
OS Platform and Distribution: RedHat 7.3
TensorFlow installed from: Customer build from source code
TensorFlow version: 1.8
Bazel version: 0.12
CUDA/cuDNN version:9.1/7.0.5
GPU model and memory: P5000/16G
Exact command to reproduce: N/A

**Comment by karmel** [comment]
Thanks, @mingyr . tf.layers.separable_conv2d is just a wrapper around tf.layers.SeparableConv2D, and tf.layers.SeparableConv1D already exists: https://www.tensorflow.org/versions/master/api_docs/python/tf/layers/SeparableConv1D . Can I suggest you use the SeparableConv1D class directly?

**Comment by tensorflowbutler** [comment]
Nagging Assignee @karmel: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
It has been 14 days with no activity and the `awaiting response` label was assigned. Is this still an issue?

**Comment by tensorflowbutler** [comment]
It has been 29 days with no activity and the `awaiting response` label was assigned. Is this still an issue?

**Comment by tensorflowbutler** [comment]
It has been 44 days with no activity and the `awaiting response` label was assigned. Is this still an issue?

**Comment by tensorflowbutler** [comment]
We are closing this issue for now due to lack of activity. Please comment if this is still an issue for you. Thanks!
